### ISSUE TITLE
Aiuto!

### ISSUE THREAD

**Issue by Matteo987** [brearlycoffee/booktab-downloader#3]
Ciao a tutti,
potreste aiutarmi? Non riesco a far partire il file "booktab-downloader.py", o meglio, lo avvio ma appare la finestra del terminale CMD per pochi millesimi di secondo.
Non sono molto esperto con il codice :)
Ringrazio fin da ora chi vorrà aiutarmi!

**Comment by Cartographic** [comment]
alla fine sei riuscito a farlo funzionare o hai rinunciato ?

**Comment by Matteo987** [comment]
> alla fine sei riuscito a farlo funzionare o hai rinunciato ?

No, non riesco. Mi sarebbe ancora utile riuscire a scaricare i libri!

**Comment by bordam** [comment]
Se qualcuno ha ancora bisogno consiglio di utilizzare [Leone25/booktab-downloader](https://github.com/Leone25/booktab-downloader)
